packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -88,26 +95,39 @@ const TradersHub = observer(() => {
                 <div id='traders-hub' className='traders-hub' ref={traders_hub_ref}>
                     <MainTitleBar />
                     <DesktopWrapper>
-                        <OrderedPlatformSections />
+                        {is_mt5_allowed ? (
+                            <OrderedPlatformSections />
+                        ) : (
+                            <OrderedPlatformSections is_cfd_visible={false} is_options_and_multipliers_visible={true} />
+                        )}
                     </DesktopWrapper>
                     <MobileWrapper>
-                        {is_landing_company_loaded ? (
-                            <ButtonToggle
-                                buttons_arr={is_eu_user ? platform_toggle_options_eu : platform_toggle_options}
-                                className='traders-hub__button-toggle'
-                                has_rounded_button
-                                is_traders_hub={window.location.pathname === routes.traders_hub}
-                                name='platforn_type'
-                                onChange={platformTypeChange}
-                                value={selected_platform_type}
-                            />
+                        {is_mt5_allowed &&
+                            (is_landing_company_loaded ? (
+                                <ButtonToggle
+                                    buttons_arr={is_eu_user ? platform_toggle_options_eu : platform_toggle_options}
+                                    className='traders-hub__button-toggle'
+                                    has_rounded_button
+                                    is_traders_hub={window.location.pathname === routes.traders_hub}
+                                    name='platforn_type'
+                                    onChange={platformTypeChange}
+                                    value={selected_platform_type}
+                                />
+                            ) : (
+                                <ButtonToggleLoader />
+                            ))}
+                        {!is_mt5_allowed && (
+                            <div className='traders-hub--mt5-not-allowed'>
+                                <Text size='s' weight='bold' color='prominent'>
+                                    <Localize i18n_default_text='Multipliers' />
+                                </Text>
+                            </div>
+                        )}
+                        {is_mt5_allowed ? (
+                            <OrderedPlatformSections />
                         ) : (
-                            <ButtonToggleLoader />
+                            <OrderedPlatformSections is_cfd_visible={false} is_options_and_multipliers_visible={true} />
                         )}
```

Comment:
You can reuse the code here because it is being repeated in 2 places.

Reply:
Sure thanks for pointing out.